randovania/server/multiplayer/world_api.py

```diff
@@ -350,6 +351,8 @@ def emit_world_pickups_update(sa: ServerApp, world: World):
                 {
                     "provider_name": action.provider.name,
                     "pickup": _base64_encode_pickup(pickup_target.pickup, resource_database),
+                    "provider_uuid": str(action.provider.uuid),
+                    "location": action.location,
```

Comment:
would be neat for these to have the same ordering as everywhere else (e.g. location first, uuid after)